lcls_tools/superconducting/scLinac.py

```diff
@@ -209,60 +206,61 @@ def move(self, numSteps: int, maxSteps: int = utils.DEFAULT_STEPPER_MAX_STEPS,
         :param changeLimits: whether or not to change the speed and steps
         :return:
         """
-
+        
         if (self.limit_switch_a_pv.value == utils.STEPPER_ON_LIMIT_SWITCH_VALUE
                 or self.limit_switch_b_pv.value == utils.STEPPER_ON_LIMIT_SWITCH_VALUE):
             raise utils.StepperError("Stepper motor on limit switch")
-
+        
         if changeLimits:
             # on the off chance that someone tries to write a negative maximum
             caput(self.max_steps_pv.pvname, abs(maxSteps), wait=True)
-
+            
             # make sure that we don't exceed the speed limit as defined by the tuner experts
             caput(self.speed_pv.pvname,
                   speed if speed < utils.MAX_STEPPER_SPEED
                   else utils.MAX_STEPPER_SPEED, wait=True)
-
+        
         if abs(numSteps) <= maxSteps:
             caput(self.step_des_pv.pvname, abs(numSteps), wait=True)
             self.issueMoveCommand(numSteps)
             self.restoreDefaults()
         else:
             caput(self.step_des_pv.pvname, maxSteps, wait=True)
             self.issueMoveCommand(numSteps)
-
+            
             self.move(numSteps - (sign(numSteps) * maxSteps), maxSteps, speed,
                       False)
-
+    
     def issueMoveCommand(self, numSteps):
-
+        
         # this is necessary because the tuners for the HLs move the other direction
         if self.cavity.cryomodule.isHarmonicLinearizer:
             numSteps *= -1
-
+        
         if sign(numSteps) == 1:
             caput(self.move_pos_pv.pvname, 1, wait=True)
         else:
             caput(self.move_neg_pv.pvname, 1, wait=True)
-
+        
         print("Waiting 5s for the motor to start moving")
         sleep(5)
-
-        while self.motor_moving_pv.value == 1:
+        
+        while self.motor_moving_pv.get() == 1:
             if self.abort_flag:
                 self.abort_pv.put(1)
                 raise utils.StepperAbortError(
-                    f"Abort requested for {self.cavity.cryomodule.name} cavity {self.cavity.number} stepper tuner")
-
-            print("Motor moving", datetime.now())
-            sleep(1)
-            if (abs(self.cavity.detune_best_PV.value) > 150000
-                    and self.cavity.freq_stop_pv.value != 400000):
+                        f"Abort requested for {self.cavity.cryomodule.name} cavity {self.cavity.number} stepper tuner")
+            
+            print(f"{self} motor moving", datetime.now())
+            
+            if abs(self.cavity.detune_best_PV.get()) > 150000:
```

Comment:
Do we just keep the chirp range at 400k? or would it make sense to make it smaller again when the cavity gets closer?

Reply:
I might just delete this because it depends on whether we're going to resonance or staying at cold landing 🙄